packages/agent/src/agent/http/index.ts

```diff
@@ -263,10 +262,28 @@ export class HttpAgent implements Agent {
     return hostname === '127.0.0.1' || hostname.endsWith('127.0.0.1');
   }
 
-  public addTransform(fn: HttpAgentRequestTransformFn, priority = fn.priority || 0): void {
-    // Keep the pipeline sorted at all time, by priority.
-    const i = this._pipeline.findIndex(x => (x.priority || 0) < priority);
-    this._pipeline.splice(i >= 0 ? i : this._pipeline.length, 0, Object.assign(fn, { priority }));
+  public addTransform(
+    type: 'call' | 'query',
```

Comment:
Maybe to check nomenclature: I thought we have:
- "update calls" and "query calls"
- "updates" and "queries"
- but not "calls" and "queries"

Reply:
I can refactor for that